### ISSUE TITLE
[ED] Gramatical changes

### ISSUE THREAD

**Issue by kakinney** [w3c/wai-intro-accessibility#21]
1. need a comma after currently: However, currently, many sites and tools are developed with accessibility barriers that make them difficult or impossible for some people to use.

2. customizable is spelled wrong: How to Meet WCAG 2.0: A customizable quick reference...
Web Accessibility Tutorials

3. no need for the comma after website: We encourage you to explore this website or look through the WAI Resources list.

Question - are we consistent with the spelling of ageing throughout our pages?    I think it should be spelled aging, but ageing is a "non-American variant" and I know that W3C is international, so I just wanted to be sure we're consistent on our pages.

**Comment by shawna-slh** [comment]
Thanks for input, Kris Anne. @yatil had some similar comments.

Here's what I'm thinking:

> need a comma after currently: However, currently, many sites and tools are developed with accessibility barriers that make them difficult or impossible for some people to use.

While that may be technically correct, I think about how I would say it, and I would not have a pause after currently. Also I think two commas in a row makes sentence more complex. 

> no need for the comma after website: We encourage you to explore this website or look through the WAI Resources list.

Agree comma not needed; however, I really want a pause there in reading it. (I feel less strong about this one, though. :)

---

> Question - are we consistent with the spelling of ageing throughout our pages? I think it should be spelled aging, but ageing is a "non-American variant" and I know that W3C is international, so I just wanted to be sure we're consistent on our pages.

Good question! Yes, we generally use US spellings; however, for this word we use "ageing". This is documented here: https://www.w3.org/WAI/EO/wiki/Style#Spelling which links to some background here https://www.w3.org/WAI/EO/wiki/Style#ageing_background

---

> customizable is spelled wrong: How to Meet WCAG 2.0: A customizable quick reference...
Web Accessibility Tutorials

fixed!

---

I'm closing this for now -- Feel free to re-open if you want more consideration on it.

**Comment by yatil** [comment]
> > need a comma after currently: However, currently, many sites and tools are developed with accessibility barriers that make them difficult or impossible for some people to use.
>
> While that may be technically correct, I think about how I would say it, and I would not have a pause after currently. Also I think two commas in a row makes sentence more complex.

The sentence is already complex. Why not say…

> Many sites and tools are currently developed with accessibility barriers which make them difficult or impossible for some people to use.

If you want to keep the however part (I didn’t have a new look at the context), you could prefix with “However,” or “But”.

**Comment by yatil** [comment]
> > Question - are we consistent with the spelling of ageing throughout our pages? I think it should be spelled aging, but ageing is a "non-American variant" and I know that W3C is international, so I just wanted to be sure we're consistent on our pages.
> 
> Good question! Yes, we generally use US spellings; however, for this word we use "ageing". This is documented here: https://www.w3.org/WAI/EO/wiki/Style#Spelling which links to some background here https://www.w3.org/WAI/EO/wiki/Style#ageing_background

I have the feeling that this is something many people stumble over, so we might want to add a footnote for the first time when we are using “ageing” on a page. I think British/Canadian/Australians are probably more stunned to see “Color vision affects ageing populations.” than “Color vision affects aging populations.”. As someone whose mother tongue is not English I find ageing lightly easier to understand because it has the full word “age” in it.

**Comment by shawna-slh** [comment]
I think we want the focus on "currently" at the beginning-- in hopes that that won't happen in the future -- ha! See [in context](https://w3c.github.io/wai-intro-accessibility/fundamentals/accessibility-intro/#context)

+1 for "ageing" easier to understand. It is for me as a native speaker as well.

I think on pages that use ageing a lot, we might want to note -(/me will look at that on older user page revisions)- but on page like this where it's one tiny point I think it would be more clutter than it's worth.

**Comment by yatil** [comment]
Re: Clutter here: Agreed.

**Comment by kakinney** [comment]
I don't think we should clutter the site with that definition since ageing is just mentioned once.  I just wanted to bring it up as a point to ask.  

Re: However, currently - then maybe change the sentence to "Currently, many sites and tools are developed with accessibility barriers that make them difficult or impossible for some people to use." and that still gives the impression that its "current" and not forever.
